api/src/MemberMatch/Member.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RaceResults.MemberMatch
+{
+    public class Member
+    {
+        public List<string> FirstList;
+        public List<string> LastList;
+        public string City;
+
+        public static List<string> ProcessName(string field)
+        {
+            /* Rules for names:
+                Assume no accent marks !!!TODO
+                trim spaces from ends !!!TODO
+                capitalize everything
+                make any middle name part of the first or last name via spaces
+                Remove "." and "'"
+                ignore any one character names
+                Split on hyphens, slashes, and spaces and treat as nicknames
+                treat the nickname column as a first name nickname
+                remove empty strings
+            */
+            field = field.ToUpperInvariant().Trim();
+
+            // TODO what about other single-quote like characters such as back quote
+            field = field.Replace(".", string.Empty).Replace("'", string.Empty);
+
+            // TODO and all whitespace?
+            string[] names = field.Split(new[] { '-', ' ', '/' }, System.StringSplitOptions.RemoveEmptyEntries);
+            var names2 =
+                (from name in names
```

Comment:
in the rest of the solution we are using the `.MethodCall()` syntax for LINQ. Would you mind switching over to that just for consistency?

Reply:
It makes me sad, because I love the LINQ syntax, but will do.

Reply:
Luke,

Can you see how you would translate this to Method Syntax and decide if it is worth it?

```
            var delta = (from coincidence in probabilityAppearsInLineByCoincidenceList
                         from isContained in isContainedList
                         from fromRef in probabilityAppearsInLineFromReferenceList
                         let altScore = Scorer.Delta(
                             probabilityAppearsInLineByCoincidence: coincidence,
                             isContained: isContained,
                             probabilityAppearsInLineFromReference: fromRef)
                         select altScore)
                        .Max();
```
(I found this hint: https://stackoverflow.com/questions/9115675/nested-from-linq-query-expressed-with-extension-methods)

Perhaps we could move simple expressions in the Method Syntax but allow the query syntax for more complex expressions.

- Carl

Reply:
It looks like you're just trying to zip these three lists together. 
```
var delta = probabilityAppearsInLineByCoincidenceList
                  .Zip(isContainedList, probabilityAppearsInLineFromReferenceList)
                  .Select(tuple => Scorer.Delta(tuple.Item1, tuple.Item2, tuple.Item3)
                  .Max();
```

Reply:
Yikes, that was also a bug -- the query syntax did nested loops accidentally. Fixed.

With the help of Resharpener, I've moved everything over to Linq Method Syntax and created a new pull request.